src/helpers/prss.rs

```diff
@@ -503,4 +513,18 @@ pub mod test {
         same_rng(rng2_l, rng1_r);
         same_rng(rng3_l, rng2_r);
     }
+
+    #[tokio::test]
+    async fn different_prss() {
+        // each time we create a step, it should generate a different prss
+        let world: TestWorld<TestStep> = make_world(QueryId);
+        let context = make_contexts(&world);
+
+        let prss_mul10 = &context[0].participant[TestStep::Mul1(0)];
+        let prss_mul11 = &context[0].participant[TestStep::Mul1(1)];
+        let prss_mul2 = &context[0].participant[TestStep::Mul2];
```

Comment:
What's Mul1 and Mul2? Why are there two "Mul" operations in "TestStep"? Could we use something named less similarly instead?

Reply:
Using a better example i.e. reshare and mul2